File: Python/Funcionario/Vendedor.py
from Funcionario import Funcionario

class Vendedor(Funcionario):
    def __init__(self, nome, cpf, salario_base, comissao):
        super().__init__(nome, cpf, salario_base)
        self.comissao = comissao
        self.salario = salario_base

    def calculaSalario(self):
        return self.salario + self.comissao

File: Java/Exercicio2/MembrosEstaticosEAbstratos/src/Vendedor.java
public class Vendedor extends Funcionario{
    private double comissao = 0;
    public Vendedor(String nome, String CPF, double salarioBase, double comissao) {
        super.setNome(nome);
        super.setCPF(CPF);
        super.setSalarioBase(salarioBase);
        this.comissao = comissao;
    }

    @Override
    public double calculaSalario() {
        return super.getSalarioBase() + this.comissao;
    }
}
